.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Count number of lines
+        run: |
+          chmod +x ./.github/workflows/scripts/countline.py
+          ./.github/workflows/scripts/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrgList/OrgList.tsx src/GraphQl/Mutations/mutations.ts src/components/EventListCard/EventListCardModals.tsx src/components/TagActions/TagActionsMocks.ts src/utils/interfaces.ts src/screens/MemberDetail/MemberDetail.tsx src/components/OrgPostCard/OrgPostCard.tsx src/components/UsersTableItem/UsersTableItem.tsx
+
+      - name: Get changed TypeScript files
+        id: changed-files
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:62: Double quote to prevent globbing and word splitting

(shellcheck)